### PR TITLE
Fall 2018 Newsletter

### PR DESCRIPTION
**Pull request by lmpeterson**
@aeslaughter  - please look at this. the last image is not showing up - broken link although the .png file seems fine.

### CODE REVIEW

**Comment by aeslaughter** [comment]
I fixed up a few things. I also have a small change going through MOOSE that removes the need for the newline you added.

**Comment by aeslaughter** [comment]
I am not sure what was going on with the PNG, I opened and saved it again and now it seems to work.

File: doc/content/news/2018-10.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# MASTODON Newsletter (Fall 2018)
+
+## MASTODON R&D 100 Finalist
+
+MASTODON is an R&D 100 finalist for 2018. The final winners will be announced on November 16th.
+
+## Beam Elements in MASTODON
+
+Geometrically nonlinear beam and lumped mass/inertia elements have been recently added to MASTODON. These elements can now be used to approximate structures using stick-mass models. As a test scenario, stick-mass model of a nuclear power plant from the SASSI manual ([image001]) was modeled using MASTODON and LS-Dyna. The first 15 natural frequencies of the MASTODON model agree well the SASSI and the corresponding LS-Dyna models, and the spectral acceleration of a location on the structure matches well with the LS-Dyna results as shown in [image002].
```
Comment (sveerara) [inline doc/content/news/2018-10.md]: Two minor changes:
1. MASTODON model agree well "with" the SASSI
2. acceleration of a location -> acceleration at a location

Comment (aeslaughter) [reply doc/content/news/2018-10.md]: @sveerara FYI, you can propose changes directly: https://help.github.com/articles/incorporating-feedback-in-your-pull-request/#applying-a-suggested-change

Comment (sveerara) [reply doc/content/news/2018-10.md]: Thanks @aeslaughter. I edited the .md file and that created a separate commit with the changes.

**Pull request merged**